client/js/store/map/ProcedureMapSettings.js

```diff
@@ -75,7 +76,7 @@ export default {
 
         const params = {
           fields: {
-            Procedure: [
+            [resourceType]: [
```

Comment:
should it be an array?

Reply:
It is not an array but a computed property name in an object. This syntax allows us to use the value of the resourceType variable as a key in the fields object.